.cspell.json

```diff
@@ -24,15 +24,17 @@
     "docs"
   ],
   "words": [
-    "Schemable",
+    "bigints",
     "combinators",
+    "nats",
+    "newtype",
+    "Purescript",
+    "Schemable",
+    "schemables",
```

Comment:
Neat! You'll have to tell me more about `cspell`, I've never used it but seems like a nice way of preventing typos.

Reply:
Yeah!  It works with the [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) vscode extension.  And we have a yarn command for it: `yarn spelling`